forage-android/src/main/java/com/joinforage/forage/android/core/services/vault/AbstractVaultSubmitter.kt

```diff
@@ -113,7 +113,6 @@ internal abstract class AbstractVaultSubmitter(
         .setHeader(ForageConstants.Headers.MERCHANT_ACCOUNT, params.merchantId)
         .setHeader(ForageConstants.Headers.IDEMPOTENCY_KEY, params.idempotencyKey)
         .setHeader(ForageConstants.Headers.TRACE_ID, logger.getTraceIdValue())
-        .setHeader(ForageConstants.Headers.API_VERSION, "default")
```

Comment:
Just calling out that we are doing this in this PR. I know it will be updated in a later PR, but we need some value set.